src/mpid/ch4/netmod/ofi/ofi_init.h

```diff
@@ -274,6 +274,7 @@ static inline int MPIDI_NM_mpi_init_hook(int rank,
 
     if (MPIDI_OFI_ENABLE_DATA) {
         hints->caps |= FI_DIRECTED_RECV;        /* Match source address    */
+        hints->domain_attr->cq_data_size = 4;   /* Minimum size for completion data entry */
```

Comment:
So @pkcoff, this is the line that's causing problems for you? Looking in the BG/Q provider's source [here](https://github.com/ofiwg/libfabric/blob/419f103501ed2605771392ac0e2cdca3b6cdf30c/prov/bgq/src/fi_bgq_domain.c#L266) I guess the value is set to `0`.

Given that the [capabilities of the provider](https://github.com/pmodels/mpich/blob/d2d5253d3031d22a7fd3a6d0d0b5692784d5f767/src/mpid/ch4/netmod/ofi/ofi_capability_sets.h#L168) are currently set to have `MPIDI_OFI_ENABLE_DATA` disabled, it's odd that this is still causing problems.